membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -103,7 +103,7 @@ class TouchpointComponents(
 
   private lazy val mapper = new SupporterRatePlanToAttributesMapper(stage)
   private lazy val dynamoSupporterProductDataService =
-    new DynamoSupporterProductDataService(dynamoClientBuilder.build(), supporterProductDataTable, mapper, createMetrics)
+    new SupporterProductDataService(dynamoClientBuilder.build(), supporterProductDataTable, mapper, createMetrics)
```

Comment:
this had a trait and a class but there was only one implementation so I collapsed them together for now